lib/sycamore/sycamore/llms/prompts/default_prompts.py

```diff
@@ -140,15 +140,15 @@ class ExtractTablePropertiesPrompt(SimplePrompt):
 
 class ExtractTablePropertiesTablePrompt(SimplePrompt):
     user = """
-            You are given a text string where columns are separated by comma representing either a single column, 
+    You are given a text string where columns are separated by comma representing either a single column, 
             or multi-column table each new line is a new row.
```

Comment:
Why not just say "You are given the CSV representation of a table."?

Reply:
yep, changed